### PR TITLE
Updated show_times description

### LINKED ISSUE #8401: cli: improve interactive sql shell timings
PR: https://github.com/cockroachdb/cockroach/pull/52233

From release notes:
> The CLI no longer prints a blanket `Time` for queries. Instead, if `show_times` is turned on and the server version is v20.2 or later, the CLI prints two separate times: the server execution time and the network latency. [#52233][#52233] {% comment %}doc{% endcomment %}

### LINKED ISSUE #8514: release-20.2: cli/sql: improve the display of statement timings
PR: https://github.com/cockroachdb/cockroach/pull/54623

From release notes:
> The timing details for query execution in the SQL shell (e.g., [`cockroach sql`](../v20.2/cockroach-sql.html), [`cockroach demo`](../v20.2/cockroach-sql.html)) have been completed to also properly take query parsing and planning time into account. As previously, this timing display can be disabled with `\set show_times off`. [#54623][#54623] {% comment %}doc{% endcomment %}

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes #8514. 
Fixes #8401.

### CODE REVIEW

File: v20.2/cockroach-sql.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ Client Options | Description
...
 `echo` | Reveal the SQL statements sent implicitly by the SQL shell.<br><br>**Default:** `false`<br><br>To enable this option, run `\set echo`. For a demonstration, see the [example](#reveal-the-sql-statements-sent-implicitly-by-the-command-line-utility) below.
 <a name="sql-option-errexit"></a> `errexit` | Exit the SQL shell upon encountering an error.<br /><br />**Default:** `false` for [interactive sessions](#session-and-output-types); `true` otherwise<br><br>To enable this option, run `\set errexit`.
-`show_times` | Reveal the time a query takes to complete.<br><br>**Default:** `true`<br><br>To disable this option, run `\unset show_times`.
+`show_times` | Reveal the time a query takes to complete. `execution` time refers to the time taken by the SQL execution engine to execute the query. `network` time refers to the network latency on the query. `other` time refers to all other forms of latency affecting the total query completion time.<br><br>**Default:** `true`<br><br>To disable this option, run `\unset show_times`.
```
Comment (knz) [inline v20.2/cockroach-sql.md:168]: network latency *between the server and the sql client command*

other forms of latency ..., including query planning.

**Review by knz — APPROVED** [review]

**Comment by ericharmeling** [comment]
TFTR @knz !

**Review by lnhsingh — COMMENTED** [review]
Content LGTM but a suggestion on formatting

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8586#-:-MJ8ZOOp7VpEpsEhKHvf:bbokvm0)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ericharmeling and @lnhsingh)

___
*[v20.2/cockroach-sql.md, line 168 at r2](https://reviewable.io/reviews/cockroachdb/docs/8586#-MJ8Z2mg1IFPLEQsvB_5:-MJ8Z2mh8N_ksiGdbp8p:b2a4wt2) ([raw file](https://github.com/cockroachdb/docs/blob/41faa115be890398b961b00b63e2c002d61cc5ee/v20.2/cockroach-sql.md#L168)):*
> ```Markdown
> `echo` | Reveal the SQL statements sent implicitly by the SQL shell.<br><br>**Default:** `false`<br><br>To enable this option, run `\set echo`. For a demonstration, see the [example](#reveal-the-sql-statements-sent-implicitly-by-the-command-line-utility) below.
> <a name="sql-option-errexit"></a> `errexit` | Exit the SQL shell upon encountering an error.<br /><br />**Default:** `false` for [interactive sessions](#session-and-output-types); `true` otherwise<br><br>To enable this option, run `\set errexit`.
> `show_times` | Reveal the time a query takes to complete. `execution` time refers to the time taken by the SQL execution engine to execute the query. `network` time refers to the network latency between the server and the SQL client command. `other` time refers to all other forms of latency affecting the total query completion time, including query planning.<br><br>**Default:** `true`<br><br>To disable this option, run `\unset show_times`.
> ```

Suggestion, make this a bulleted list. For example:

```
Reveal the time a query takes to complete. Possible values:
- `execution` time refers to...
- `network` time refers to...
-  `other` ...
```

**Review by ericharmeling — COMMENTED** [review]
TFTR @lnhsingh 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8586#-:-MJ8bXl_4jDn4oUcNjAt:byxoliw)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @knz and @lnhsingh)

___
*[v20.2/cockroach-sql.md, line 168 at r1](https://reviewable.io/reviews/cockroachdb/docs/8586#-MJ3WYiGlgGRhdLXxCWZ-r1-168:-MJ8bWXvCd2GaXAzFnZr:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/ba41cd9e461ef9d76d5bc57aef46e9c3bd8844ae/v20.2/cockroach-sql.md#L168)):*
<details><summary><i>Previously, knz (kena) wrote…</i></summary><blockquote>

network latency *between the server and the sql client command*

other forms of latency ..., including query planning.
</blockquote></details>

Done.
___
*[v20.2/cockroach-sql.md, line 168 at r2](https://reviewable.io/reviews/cockroachdb/docs/8586#-MJ8Z2mg1IFPLEQsvB_5:-MJ8bWvE6XHxjtBg2jXi:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/41faa115be890398b961b00b63e2c002d61cc5ee/v20.2/cockroach-sql.md#L168)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

Suggestion, make this a bulleted list. For example:

```
Reveal the time a query takes to complete. Possible values:
- `execution` time refers to...
- `network` time refers to...
-  `other` ...
```
</blockquote></details>

Done.

**Pull request merged**